### ISSUE TITLE
Improve aggregated_metadata.json ordering

### ISSUE THREAD

**Issue by davidsmejia** [AlexsLemonade/refinebio#3459]
### Context

When we smash datasets together and create the `aggreggated_metadata.json` file, we set the keys in alphabetical order.

While this has no impact of the performance one way or the other, it is not unreasonable for a human person to want to review this file by eye and this ordering might not be the best because of the deeply nested values.

### Problem or idea

`aggregated_metadata.json` is currently serialized in alphabetical order:

```js
  aggregate_by // download option
  created_at
  experiments
  ks_pvalue
  ks_statistic
  ks_warning
  num_experiments
  num_samples
  quant_sf_only // download option
  quantile_normalized // download option
  samples
  scale_by // download option
```

### Solution or next step

I think we should place the values that are atomic at the top and then experiments / samples at the bottom.

I am less fond of the idea of nesting all download options together as this would be a breaking change for automated workflows.
  
@jaclyn-taroni for input

**Comment by jaclyn-taroni** [comment]
> I think we should place the values that are atomic at the top and then experiments / samples at the bottom.

I agree with this approach. Good job anticipating my suggestion about nesting download options together, as well 😄
